desmume/src/cheatSystem.h

```diff
@@ -120,6 +120,9 @@ class CHEATS
 	
 	bool	remove(const size_t pos);
 	
+	void	toggle(bool enabled, const size_t pos);
+	void	toggle(u8 enablbed, const size_t pos);

```

Comment:
typo. do we really need an overloaded func for this ?

Reply:
Maybe not, but the entire cheatSystem overloaded `enabled` this way: `add` [L99-100](https://github.com/TASEmulators/desmume/blob/058fc066c521e1e5f6218a8fcbc211fc37204a74/desmume/src/cheatSystem.h#L99-L100), `update` [L103-104](https://github.com/TASEmulators/desmume/blob/058fc066c521e1e5f6218a8fcbc211fc37204a74/desmume/src/cheatSystem.h#103-L104), `add_AR` [L109-110](https://github.com/TASEmulators/desmume/blob/058fc066c521e1e5f6218a8fcbc211fc37204a74/desmume/src/cheatSystem.h#L109-L110), etc. I followed suit, assuming there was a design reason behind it.

Reply:
then it's probably fine.